### PR TITLE
New version: vatis-project.vAtis version 4.1.0-beta.1

### LINKED ISSUE #200510: [Update Request]: vatis-project.vAtis version 4.1.0-beta.1
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

vatis-project.vAtis

### Package Version

4.1.0-beta.1

### Please describe the changes you would like to see

~~昨天还好好的怎么今天连仓库都没了...~~  
I found a new version of this package at https://vatis.app/ This package should be updated from version 4.0.0 to 4.1.0.

```[tasklist]
### Pull request for removal of version 4.0.0 of this package
- [ ] #200517
- [ ] #200026
- [ ] #200518
- [ ] #200520 
- [ ] #200521 
- [ ] #200522 
- [ ] #200524 
- [ ] #200525 
- [ ] #200526 
- [ ] #200527 
- [ ] #200528 
- [ ] #200530 
- [ ] #200531 
- [ ] #200532 
```

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200510

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200901)

### CODE REVIEW

File: manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: vatis-project.vAtis
+PackageVersion: 4.1.0-beta.1
+PackageLocale: en-US
+Publisher: vATIS for VATSIM
+PublisherUrl: https://github.com/vatis-project
+PublisherSupportUrl: https://github.com/vatis-project/vatis-archive/issues
```
Comment (Dragon1573) [inline manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml:9]:
```suggestion
```

Issues is disabled in their repository.

File: manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+PackageName: vATIS
+PackageUrl: https://github.com/vatis-project/vatis-archive
```
Comment (Dragon1573) [inline manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml:11]:
```suggestion
PackageUrl: https://github.com/vatis-project/vatis
```

They migrated to a [new repository path](https://github.com/vatis-project/vatis).

File: manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+License: GPL-3.0
+LicenseUrl: https://github.com/vatis-project/vatis-archive/blob/HEAD/LICENSE
```
Comment (Dragon1573) [inline manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml:13]:
```suggestion
LicenseUrl: https://github.com/vatis-project/vatis/blob/main/LICENSE
```

(See previous comments)

File: manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: vatis-project.vAtis
+PackageVersion: 4.1.0-beta.1
+PackageLocale: en-US
+Publisher: vATIS for VATSIM
+PublisherUrl: https://github.com/vatis-project
```
Comment (Dragon1573) [inline manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://github.com/vatis-project
PublisherSupportUrl: https://discord.gg/rTMZhjhu9Q
```

Extracted from [Getting Help | vATIS for VATSIM](https://vatis.app/docs/getting-help)

**Review by Dragon1573 — COMMENTED** [review]
Missing of `ReleaseNotes` and `ReleaseNotesUrl` is intended, as I can't find any one of them in their homepage.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**